src/fetch.py

```diff
@@ -89,6 +100,98 @@ def read_json(self, file):
         except IOError:
             return False
 
+    def get_server_version(self):
+        """Get version of server which is being processed
+
+        Note that the path to the version is the same across Ping Federate
+        versions."""
+        api_path = f'{self.ping_data["basePath"]}/version'
```

Comment:
I'm getting the following error on my end:
```
  File "/Users/teddevaal/dev/ping/py-ping-fed-sdk/src/fetch.py", line 108, in get_server_version
    api_path = f'{self.ping_data["basePath"]}/version'
KeyError: 'basePath'
```

Reply:
Ah - the `get_server_version` call should have been in the Swagger 2.0 specific part (because only 2.0 specs have the `basePath` key).

Fixed.